swift/apple/FirezoneKit/Sources/FirezoneKit/Managers/TunnelManager.swift

```diff
@@ -93,21 +93,19 @@ public class TunnelManager {
   // Encoder used to send messages to the tunnel
   private let encoder = PropertyListEncoder()
 
-  // Use separate bundle IDs for release and debug.
-  // Helps with testing releases and dev builds on the same Mac.
-  #if DEBUG
-    private let bundleIdentifier = Bundle.main.bundleIdentifier.map {
-      "\($0).debug.network-extension"
-    }
+#if os(macOS)
+  private let systemExtensionManager: SystemExtensionManager?
+#endif
 
-    private let bundleDescription = "Firezone (Debug)"
-  #else
-    private let bundleIdentifier = Bundle.main.bundleIdentifier.map { "\($0).network-extension" }
-    private let bundleDescription = "Firezone"
-  #endif
+  private let bundleIdentifier = "\(Bundle.main.bundleIdentifier!).network-extension"
+  private let bundleDescription = "Firezone"
```

Comment:
This was simplified because SystemExtensions aren't as sensitive to conflicting bundle IDs as App Extensions.